src/Layers/xrRenderDX10/dx10HW.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include "Layers/xrRender/HWCaps.h"
+#include "xrCore/ModuleLookup.hpp"
+#include "SDL.h"
+#include "SDL_syswm.h"
+
+#if !defined(_MAYA_EXPORT)
+#include "Layers/xrRender/stats_manager.h"
+#endif
+
+class CHW
+    : public pureAppActivate,
+      public pureAppDeactivate
+{
+public:
+    CHW();
+    ~CHW();
+
+
+    void CreateD3D();
+    void DestroyD3D();
+
+    void CreateDevice(SDL_Window* sdlWnd);
+
+    void CreateSwapChain(HWND hwnd);
+    bool CreateSwapChain2(HWND hwnd);
+
+    void DestroyDevice();
+
+    void Reset();
+
+    D3DFORMAT selectDepthStencil(D3DFORMAT);
+    u32 selectPresentInterval();
+    u32 selectGPU();
+    BOOL support(D3DFORMAT fmt, DWORD type, DWORD usage);
```

Comment:
You can safely remove `support` and `selectDepthStencil` functions from dx10HW.

Reply:
`selectGPU()` can also be safely removed.

Reply:
`support` is used in R4 by `CRender::create()` when checking for DF24 format availability

Reply:
It's a rudimentary code from DX9 renderer, which can be safely removed too.
Explanation:
![image](https://user-images.githubusercontent.com/2989212/64132987-e2a5bf80-cdec-11e9-8ba5-caaefc2c4041.png)